src/Microsoft.AspNet.Mvc.Core/Rendering/Internal/TemplateRenderer.cs

```diff
@@ -7,16 +7,19 @@
 using System.Globalization;
 using System.IO;
 using System.Linq;
+using Microsoft.AspNet.Http;
 using Microsoft.AspNet.Mvc.Core;
+using Microsoft.AspNet.Mvc.ModelBinding;
 using Microsoft.Framework.DependencyInjection;
 using Microsoft.Framework.Internal;
 
-namespace Microsoft.AspNet.Mvc.Rendering
+namespace Microsoft.AspNet.Mvc.Rendering.Internal
 {
-    internal class TemplateRenderer
+    public class TemplateRenderer
     {
         private static readonly string DisplayTemplateViewPath = "DisplayTemplates";
         private static readonly string EditorTemplateViewPath = "EditorTemplates";
+        public static readonly string IEnumerableOfIFormFileName = "IEnumerable`" + nameof(IFormFile);
```

Comment:
I am confused. I changed this from `const` based on the feedback here. https://github.com/aspnet/Mvc/pull/2366#discussion_r28301603